x-pack/test/reporting_functional/reporting_and_security.config.ts

```diff
@@ -24,7 +24,6 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       serverArgs: [
         ...functionalConfig.get('kbnTestServer.serverArgs'),
         `--xpack.reporting.capture.maxAttempts=1`,
-        `--xpack.reporting.csv.maxSizeBytes=6000`,
```

Comment:
I think it makes more sense to keep this setting stuck with the default for all types of reporting tests.